website/docs/cloud-docs/integrations/kubernetes/k8s-ops-v2-migration.mdx

```diff
@@ -0,0 +1,104 @@
+---
+page_title: Terraform Cloud Kubernetes Operator v2 Migration Guide
+description: >-
+  Upgrade the Terraform Cloud Kubernetes Operator from version 1 to version 2.
+tfc_only: false
+---
+
+# Terraform Cloud Kubernetes Operator v2 Migration Guide
+
+To upgrade the Terraform Cloud Kubernetes Operator from version 1 to version 2, there is a one-time process that you need to complete. This process will upgrade the operator to the newest version and migrate your custom resources. 
+
+## Prepare for the upgrade
+
+Before you migrate to Terraform Cloud Kubernetes Operator v2, you must first update v1 of the operator to the latest version, including the custom resource definitions.
+
+```shell-session
+$ helm upgrade --namespace ${RELEASE_NAMESPACE} ${RELEASE_NAME} hashicorp/terraform
+```
+
+Next, backup the workspace resources.
+
+```shell-session
+$ kubectl get workspace --all-namespaces -o yaml > backup_tfc_operator_v1.yaml
+```
+
+Finally, we suggest that you set the `applyMethod` of each workspace to `manual`. In version 2 of the Terraform Cloud Kubernetes Operator this is the default value, however explicitly configuring it with the value provides extra protection from unintended automatic applies during the upgrade process.
+
+```shell-session
+kubectl patch workspace <WORKSPACE_NAME> --type=merge --patch '{"spec": {"applyMethod": "manual"}}'
```

Comment:
So they run this for each of their TFC workspaces? Or the ones that use kubernetes? Can we give them more specific guidance here?

Reply:
Or does this refer to kubernetes workspaces?